packages/server/graphql/intranetSchema/mutations/setNewTeamLead.ts

```diff
@@ -0,0 +1,62 @@
+import {GraphQLBoolean, GraphQLID, GraphQLNonNull} from 'graphql'
+import getRethink from '../../../database/rethinkDriver'
+import {isTeamLead, requireSU} from '../../../utils/authorization'
+import {InternalContext} from '../../graphql'
+import GraphQLEmailType from '../../types/GraphQLEmailType'
+import {getUserByEmail} from '../../../postgres/queries/getUsersByEmails'
+
+const setNewTeamLead = {
+  type: GraphQLNonNull(GraphQLBoolean),
+  description: 'Sets given team member as a team leader. ',
+  args: {
+    teamId: {
+      type: GraphQLNonNull(GraphQLID),
+      description: 'Team id of the team which is about to get a new team leader'
+    },
+    newTeamLeadEmail: {
+      type: GraphQLNonNull(GraphQLEmailType),
+      description: 'Email of the user who will be set as a new team leader'
+    }
+  },
+  resolve: async (
+    _source,
```

Comment:
please add type `_source: unknown`. `unknown` is a safe type for unused variables because it doesn't allow you to do anything with it